docker/Dockerfile-dhvirtualenv

```diff
@@ -64,6 +64,7 @@ ENV distro ${distro}
 
 # Python < 3.7 assumes LANG="C" means ASCII-only and throws on printing unicode
 # http://bugs.python.org/issue19846
+# TODO: We can possibly remove this now that we don't support Python 3.7
 ENV LANG C.UTF-8
```

Comment:
Can we remove this now?

I think we could have removed it when we dropped Python 3.6 support since this was defaulted to UTF-8 in Python 3.7

Reply:
Removed :white_check_mark: